Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -660,6 +660,7 @@ struct FrameHeader {
     FixedArray<u8> ec_upsampling {};
 
     u8 group_size_shift { 1 };
+    u16 group_dim { static_cast<u16>(128 << group_size_shift) };
```

Comment:
Looks like this never set to something differentn from `128 << group_size_shift` – should this be a method that does `u16 group_dim() const { return static_cast<u16>(128 << group_size_shift); }`?

(also, `group_dim` seems a un-serenity-y name. No abbreviations, etc.)

Reply:
Having it as a property is way better, thanks! This is again a name from the spec.